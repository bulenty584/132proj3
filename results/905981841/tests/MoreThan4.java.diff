6a7,12
> 6
> 5
> 4
> 3
> 2
> 1
