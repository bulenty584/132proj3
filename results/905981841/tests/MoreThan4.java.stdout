func Main()
  y0 = 0
  z0 = 4
  z1 = alloc(z0)
  z2 = 8
  vmt_MT4 = alloc(z2)
  z3 = @MT4_Start
  [vmt_MT4 + 0] = z3
  z4 = @MT4_Change
  [vmt_MT4 + 4] = z4
  [z1 + 0] = vmt_MT4
  if0 z1 goto z1Error_0
  goto z1End_1
z1Error_0:
  error("null pointer")
z1End_1:
  if0 z1 goto z1Error_2
  goto z1End_3
z1Error_2:
  error("null pointer")
z1End_3:
  z5 = 1
  z6 = 2
  z7 = 3
  z8 = 4
  z9 = 5
  z10 = 6
  z11 = [z1 + 0]
  z12 = [z11 + 0]
  z13 = call z12(z1 z5 z6 z7 z8 z9 z10)
  print(z13)
  return y0
func MT4_Start(this p1 p2 p3 p4 p5 p6)
  print(p1)
  print(p2)
  print(p3)
  print(p4)
  print(p5)
  print(p6)
  if0 this goto thisError_4
  goto thisEnd_5
thisError_4:
  error("null pointer")
thisEnd_5:
  z14 = [this + 0]
  z15 = [z14 + 4]
  z16 = call z15(this p6 p5 p4 p3 p2 p1)
  aux = z16
  return aux
func MT4_Change(this p1 p2 p3 p4 p5 p6)
  print(p1)
  print(p2)
  print(p3)
  print(p4)
  print(p5)
  print(p6)
  z17 = 0
  return z17
